React/wbooks/src/app/screens/BookDetail/components/Comment/index.js

```diff
@@ -1,24 +1,39 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
 
-import profilePicture from '../../../../assets/profile_picture.png';
+import Loader from '../../../../components/Loader';
 
-import styles from './styles.scss';
+import Comment from './layout';
 
-const Comment = () => (
-  <div className={styles.commentContainer}>
-    <div className={styles.pictureContainer}>
-      <img src={profilePicture} className={styles.profilePicture} alt="profilePicture" />
-    </div>
-    <div className={styles.commentDetail}>
-      <h1 className={styles.commentTitle}>Perez Franco</h1>
-      <h2 className={styles.commentDate}> xx/xx/xx </h2>
-      <span className={styles.commentDescription}>
-        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore
-        et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-        aliquip ex ea commodo consequat.
-      </span>
-    </div>
-  </div>
+const CommentContainer = props => (
+  <div>{props.comments.map(comment => <Comment key={comment.id} comment={comment} />)}</div>
 );
 
-export default Comment;
+const getLastComments = createSelector([state => state.books.comments], comments => {
+  if (comments) {
+    if (comments.length > 3) {
+      return comments.slice(0, 4);
+    }
+    return comments;
+  }
+  return null;
+});
+
+const mapStateToProps = state => ({
+  comments: getLastComments(state),
+  isLoading: state.books.commentsLoading
+});
+
+CommentContainer.propTypes = {
+  comments: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.number.isRequired,
+      content: PropTypes.string.isRequired,
+      created_at: PropTypes.string.isRequired
+    })
+  )
+};
+
+export default connect(mapStateToProps)(Loader(CommentContainer));
```

Comment:
You shouldn't render a entire screen spinner where u are fetching comments. If u want, u can render a spinner in the send button, but that is not in the scope of the card